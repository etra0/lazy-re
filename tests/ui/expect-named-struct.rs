use lazy_re::{lazy_re, LazyRe};

#[repr(C, packed)]
#[derive(LazyRe)]
#[lazy_re]
struct Bar(u32, u32, i8);

#[lazy_re]
enum Baz {
    a, b, c
}

#[lazy_re]
union Bax {
    a: usize,
    b: u64,
}

#[lazy_re]
pub fn foo() {}

fn main() {}
