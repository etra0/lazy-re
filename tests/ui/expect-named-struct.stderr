error: Expected named struct
 --> tests/ui/expect-named-struct.rs:6:8
  |
6 | struct Bar(u32, u32, i8);
  |        ^^^

error: Expected named struct
 --> tests/ui/expect-named-struct.rs:9:6
  |
9 | enum Baz {
  |      ^^^

error: Expected named struct
  --> tests/ui/expect-named-struct.rs:14:7
   |
14 | union Bax {
   |       ^^^

error: expected one of: `struct`, `enum`, `union`
  --> tests/ui/expect-named-struct.rs:20:5
   |
20 | pub fn foo() {}
   |     ^^
